pr.ts

```diff
@@ -26,16 +26,20 @@ export const changelog = wrap("Require changelog entries on PRs with code change
   const getContentParams = { path: "", owner: pr.head.user.login, repo: pr.head.repo.name }
   const rootContents: any = await danger.github.api.repos.getContent(getContentParams)
 
-  const hasChangelog = rootContents.data.find(file => changelogs.includes(file.name))
-  if (hasChangelog) {
+  const hasChangelog = rootContents.data.find((file: any) => changelogs.includes(file.name))
+  const markedTrivial = pr.title.match("#trivial")
```

Comment:
Title only or body as well? (meaning either)

On Danger I have a rule for title or body and I usually find myself adding it in the body itself as well. I think it's a bit less intrusive... Adding "Flags" to the title kind dirties up the issues log

Reply:
I actually prefer using the title _because_ it gets included in the log (we use this at Artsy and when looking through commits, it's nice to immediately know that a PR bypassed rules like this one). But adding it to the body as well is pretty easy and I don't see any harm in it, so let's go ahead with that.

Reply:
@ashfurrow This is just the PR title, I don't think it appears in the log anyways... 

Anyways, if it doesn't have any negative effect or is too hard I personally prefer it. I prefer leaving room in the PR title as it's relatively short, and adding flags to the end of it would just have them truncated mostly (or not leave enough room for the actual PR title)

Reply:
I agree – the RxSwiftCommunity has many ways of running our projects, and we should make this as applicable to contributors workflows as possible. I added this in https://github.com/RxSwiftCommunity/peril/pull/1/commits/8276c8101ec3ba34ee2103b4f575f73ee0f397d7 .